hugolib/hugo_sites_test.go

```diff
@@ -659,6 +670,106 @@ func TestChangeDefaultLanguage(t *testing.T) {
 	assertFileContent(t, "public/sect/doc2/index.html", true, "Single", "Hello")
 }
 
+func TestTableOfContentsInShortcodes(t *testing.T) {
+	testCommonResetState()
+
+	sites := createMultiTestSites(t, testSiteConfig{DefaultContentLanguage: "en"}, multiSiteTOMLConfigTemplate)
+
+	writeSource(t, "layouts/shortcodes/toc.html", tocShortcode)
+	writeSource(t, "content/post/simple.en.md", tocPageSimple)
+	writeSource(t, "content/post/withSCInHeading.en.md", tocPageWithShortcodesInHeadings)
+
+	cfg := BuildCfg{}
+
+	err := sites.Build(cfg)
+
+	if err != nil {
+		t.Fatalf("Failed to build sites: %s", err)
+	}
+
+	// Not checking the heading-ids because they are not generated by the ToC.
+	assertFileContentRegexp(t, "public/en/post/simple/index.html", true, tocPageSimpleExpected)
```

Comment:
Ok, I just tried adding `plainIDAnchors = true` to the `[blackfriday]` section of the config but it didn't change anything. According to the [documentation](https://gohugo.io/overview/configuration/) it is true by default and it isn't what we are looking for.

My idea was that my test should not break when/if the id-generation is altered. I have found a different sollution for that. My test is now using the `#heading {#id}` method of secifying ids.